uvicorn/workers.py

```diff
@@ -16,7 +17,7 @@ class UvicornWorker(Worker):
 
     CONFIG_KWARGS = {"loop": "uvloop", "http": "httptools"}
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
Suggested: `*args: tuple, **kwargs: dict`

Reply:
https://www.python.org/dev/peps/pep-0484/#arbitrary-argument-lists-and-default-argument-values